pkg/server/http_handler.go

```diff
@@ -0,0 +1,101 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+
+	// "github.com/dose-na-nuvem/customers/pkg/model"
+	// "github.com/dose-na-nuvem/customers/pkg/telemetry"
+	// "go.opentelemetry.io/otel/attribute"
+	// "go.opentelemetry.io/otel/trace"
+	"go.uber.org/zap"
+)
+
+var _ http.Handler = (*TollStationHandler)(nil)
+
+// type CustomerStore interface {
+// 	CreateCustomer(string) (*model.Customer, error)
+// 	ListCustomers() ([]model.Customer, error)
+// }
+
+type TollStationHandler struct {
+	logger *zap.Logger
+	// store  CustomerStore
+}
+
+func NewTollStationHandler(logger *zap.Logger /*, store CustomerStore*/) *TollStationHandler {
+	return &TollStationHandler{
+		logger: logger,
+		//store:  store,
+	}
+}
+
+func (h *TollStationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+
+	b, err := json.Marshal(true)
```

Comment:
Dica: evite constantes mágicas.

```suggestion
	openGate := true
	b, err := json.Marshal(openGate)
```